Crash on clicking Filter button for Activity view

# Bug Report

## Description

Crash on clicking Filter button for Activity view

```
panic: interface conversion: interface {} is uint64, not int64

goroutine 70096 [running]:
github.com/status-im/status-go/services/wallet/activity.(*Service).GetOldestTimestampAsync.func2({0x10ed48320?, 0x14017c82110}, {0x0?, 0x0?}, {0x0, 0x0})
	/Users/jenkins/workspace/status-desktop/systems/macos/aarch64/package/vendor/status-go/services/wallet/activity/service.go:320 +0x12c
github.com/status-im/status-go/services/wallet/async.(*Scheduler).finishedTask(0x140004cb920, {0x10ed48320, 0x14017c82110}, 0x14013a1e270, 0x1400395a930, {0x0, 0x0})
	/Users/jenkins/workspace/status-desktop/systems/macos/aarch64/package/vendor/status-go/services/wallet/async/scheduler.go:180 +0x3a8
github.com/status-im/status-go/services/wallet/async.(*Scheduler).Enqueue.func3({0x10ed48320?, 0x14017c82110?}, 0x10f1f0200?, {0x0?, 0x0?})
	/Users/jenkins/workspace/status-desktop/systems/macos/aarch64/package/vendor/status-go/services/wallet/async/scheduler.go:128 +0x50
github.com/status-im/status-go/services/wallet/async.(*Scheduler).runTask.func1()
	/Users/jenkins/workspace/status-desktop/systems/macos/aarch64/package/vendor/status-go/services/wallet/async/scheduler.go:149 +0x104
created by github.com/status-im/status-go/services/wallet/async.(*Scheduler).runTask
	/Users/jenkins/workspace/status-desktop/systems/macos/aarch64/package/vendor/status-go/services/wallet/async/scheduler.go:140 +0x158
SIGABRT: Abnormal termination.
zsh: abort      ./nim_status_client --datadir=~/data/29_1 --LOG_LEVEL=DEBUG
```

## Steps to reproduce

1. open wallet
2. open activity tab
3. click filter

### Additional Information

- Status desktop version: **release 2.29.0 rc 7**
- Operating System: macOS

--- a/services/wallet/activity/service.go
+++ b/services/wallet/activity/service.go
@@ -317,7 +317,7 @@ func (s *Service) GetOldestTimestampAsync(requestID int32, addresses []common.Ad
 		if errors.Is(err, context.Canceled) || errors.Is(err, async.ErrTaskOverwritten) {
 			res.ErrorCode = ErrorCodeTaskCanceled
 		} else if err == nil {
-			res.Timestamp = result.(int64)
+			res.Timestamp = int64(result.(uint64))
 			res.ErrorCode = ErrorCodeSuccess
 		}
 

